travis.sbt

```diff
@@ -0,0 +1,32 @@
+commands ++= Seq(
+  // check policies, if on master also upload
+  Command.command("whitesourceCi") { state =>
+    val extracted = Project.extract(state)
+    import extracted._
+
+    if (sys.env.getOrElse("TRAVIS_SECURE_ENV_VARS", "") == "true") {
+      val stateWithCredentials =
+        append(credentials += Credentials("whitesource", "whitesourcesoftware.com", "", System.getenv("WHITESOURCE_KEY")), state)
```

Comment:
I think this is no longer needed since https://github.com/lightbend/sbt-whitesource/pull/29